### ISSUE TITLE
automatic installer for forge crashes again in v2.2

### ISSUE THREAD

**Issue by evschlenz** [cdbbnnyCode/modpack-installer#11]
The automatic installer crashes again in v2.2.

I used `python3 install.py NonFinalConfiglessFix.zip` with [this](https://www.curseforge.com/minecraft/modpacks/test-mod-cl/download/2885168) modpack and got this output:
```
[pack data already unzipped]
[minecraft dir already created]
Installing modloader
Updating user launcher profiles
Attempting to launch v2 installer
using v0 Install class
Exception in thread "main" java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
        at ForgeHack.run_v2(ForgeHack.java:146)
        at ForgeHack.main(ForgeHack.java:184)
Caused by: java.lang.NullPointerException: Cannot invoke "String.startsWith(String)" because "name" is null
        at java.base/java.lang.Class.resolveName(Class.java:3119)
        at java.base/java.lang.Class.getResourceAsStream(Class.java:2814)
        at net.minecraftforge.installer.json.Util.loadVersion(Util.java:53)
        at net.minecraftforge.installer.actions.Action.<init>(Action.java:49)
        at net.minecraftforge.installer.actions.ClientInstall.<init>(ClientInstall.java:37)
        ... 7 more
Error running the auto-installer, try using --manual.

```

Tested on Ubuntu 21.04 with Java 17 (openjdk version 17.0.1 2021-10-19)

**Comment by cdbbnnyCode** [comment]
Interesting, it looks like it's crashing *in* the Forge installer code.

**Comment by cdbbnnyCode** [comment]
This specific installer seems to have code for two entirely different installer versions (the later version being broken). Since my code looks at latest versions first, it encountered the new (broken) code and proceeded to crash. Rearranging version checks should fix this.

**Comment by cdbbnnyCode** [comment]
Fixed in commit 7755c67.
